### DISCUSSION TITLE
Patch Size Information

### DISCUSSION THREAD

**Discussion by Vandenbg** [idaholab/moose#24663] [Q&A General]
Hello, I am looking for more information on Patch Size used within the Mesh block. I am not able to find much information or any at all explaining what this parameter is. I have found only this information: 
https://mooseframework.inl.gov/source/mesh/FileMesh.html
https://mooseframework.inl.gov/syntax/Mesh/index.html

**Comment by GiudGiud** [comment]
Hello
That's all we have. You can dive into the code to find out more.
I've never used it

Guillaume
